_shared/hooks/post_gen_project.py

```diff
@@ -20,7 +20,32 @@ def remove_conditional_files():
     {% endif %}
 
     {% if cookiecutter.get("postgres") != "yes" %}
-    paths_to_remove.extend(["bin/create-db", "bin/make_db", "conf/alembic.ini"])
+    paths_to_remove.extend([
+        "bin/create-db",
+        "bin/make_db",
+        "conf/alembic.ini",
+        "{{ cookiecutter.package_name }}/db.py",
+        "{{ cookiecutter.package_name }}/migrations/env.py",
+        "{{ cookiecutter.package_name }}/migrations/script.py.mako",
+        "{{ cookiecutter.package_name }}/models/__init__.py",
+        "tests/factories/factoryboy_sqlalchemy_session.py",
+        "tests/unit/{{ cookiecutter.package_name }}/models/__init__.py",
+        "tests/unit/{{ cookiecutter.package_name }}/models/__init___test.py",
+    ])
+    {% endif %}
```

Comment:
This PR adds lots of new files that're only relevant when the `postgres` option is enabled, this makes sure that we don't add those files to projects that don't use the `postgres` option